src/org/zaproxy/zap/extension/spider/SpiderAPI.java

```diff
@@ -289,15 +290,17 @@ private GenericScanner2 getSpiderScan(JSONObject params) throws ApiException {
 	 * The {@code scanIdCounter} is used to generate the ID of the started spider scan. The started {@code SpiderScan} is saved
 	 * in {@code spiderScans} for later access/control, accessible with the returned ID.
 	 * </p>
-	 * 
+	 *
 	 * @param url the url to start the spider scan
 	 * @param user the user to scan as, or null if the scan is done without the perspective of any user
+	 * @param maxChildren Max number of children to scan
+	 * @param recurse Whether or not to scan recursively
```

Comment:
Good question, I'm not sure about the name but I think it should contain "seeds" (in the end it's what is controlling), "scanChildren" doesn't sound good either (even though that's being used in spider itself), because children pages will still be scanned/spidered when false.

Probably better to leave the rename to other issue, if we can't find a better name in the meantime.